en/components/general/update-guide.md

```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
+    For each theme included in Ignite UI for Angular we provide specific `font-family` and `type-scale` variables which you can use:
 
     | **Theme** | **Font Family** | **Type Scale** |
     |----------------|-----------------|-----------------|
     | Material | $material-typeface | $material-type-scale |
     | Fluent | $fluent-typeface | $fluent-type-scale |
     | Bootstrap | $bootstrap-typeface | $bootstrap-type-scale |
     | Indigo | $indigo-typeface | $indigo-type-scale |
- 
+
+### `IgxBottomNav` and `IgxTabs`
+The two components were completely refactored in order to provide more flexible and descriptive way to define tab headers and contents. It is recommended that you update via **ng update** in order to migrate the existing **igx-tabs** and **igx-bottom-nav** definitions to the new ones.
+
+* Template
+    * The new structure defines tab items each wrapping a header and a content. The headers may contain a label and/or an icon as well as any other custom content.
+    * For header styling purposes we introduced four new directives, namely `IgxBottomNavHeaderLabel`, `IgxBottomNavHeaderIcon`, `IgxTabHeaderLabel` and `IgxTabHeaderIcon`.
+    * Since the header component now allows adding any content, the `IgxTab` directive, which was previously used to retemplate the tab's header, was removed because it is no longer necessary.
+
+    ```html
+    <igx-bottom-nav>
+        <igx-bottom-nav-item>
+            <igx-bottom-nav-header>
+                <igx-icon igxBottomNavHeaderIcon>folder</igx-icon>
+                <span igxBottomNavHeaderLabel>Tab 1</span>
+            </igx-bottom-nav-header>
+            <igx-bottom-nav-content>
+                Content 1
+            </igx-bottom-nav-content>
+        </igx-bottom-nav-item>
+        ...
+    </igx-bottom-nav>
+
+    <igx-tabs>
+        <igx-tab-item>
+            <igx-tab-header>
+                <igx-icon igxTabHeaderIcon>folder</igx-icon>
+                <span igxTabHeaderLabel>Tab 1</span>
+            </igx-tab-header>
+            <igx-tab-content>
+                <h1>Tab 1 Content</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+            </igx-tab-content>
+        </igx-tab-item>
+    ...
+    </igx-tabs>
+    ```
+* The **API** of the components was changed during the refactoring.
+    * The `id`, `groups`, `viewTabs`, `contentTabs` and `tabs` properties were removed. Currently, the `items` property returns the collection of tabs.
+    * The following properties were changed:
+        * The tab item's `isSelected` was renamed to `selected`.
+    * The `tabItemSelected` and `tabItemDeselected` events were removed. We introduced three new events, `selectedIndexChanging`, `selectedIndexChange` and `selectedItemChange`, which provide more flexibility and control over the tabs' selection. Unfortunately, having an adequate migration for these event changes is complicated to say the least, so any errors should be handled at project level.
```

Comment:
IMO We should split those for the two components. I am not sure those were called the same for the tabs and bottom nav components. For example the events of the bottom nav were called `onTabSelected`/`onTabDeselected`.